relayer/spy_relayer/src/backends/default/listener.ts

```diff
@@ -232,9 +232,8 @@ export class TokenBridgeListener implements Listener {
     const originChain = parsedVaa.payload.originChain;
     const originAddress = parsedVaa.payload.originAddress;
 
-    let originAddressNative: string;
     try {
-      originAddressNative = tryHexToNativeString(originAddress, originChain);
+      tryHexToNativeString(originAddress, originChain);
```

Comment:
Why isn’t the assignment needed here but the function call is?

Reply:
Resolved - I just removed the check since it's redundant - it occurs in the validate function called previously